pkg/filter/filter.go

```diff
@@ -87,8 +87,16 @@ func (f *Filter) ShouldIgnoreDMLEvent(ts uint64, schema, table string) bool {
 
 // ShouldIgnoreDDLEvent removes DDLs that's not wanted by this change feed.
 // CDC only supports filtering by database/table now.
-func (f *Filter) ShouldIgnoreDDLEvent(ts uint64, schema, table string) bool {
-	return f.shouldIgnoreStartTs(ts) || f.ShouldIgnoreTable(schema, table)
+func (f *Filter) ShouldIgnoreDDLEvent(ts uint64, ddlType model.ActionType, schema, table string) bool {
+	var shouldIgnoreTableOrSchema bool
+	switch ddlType {
```

Comment:
what should we do with `RenameTable`

Reply:
just skip the DDL and DMLs after this. what do you think about it?

Reply:
case 1:
- rules = ['test.t1'], user create table `test`.`t1` in upstream, `test`.`t1` is replicated.
- rename `t1` to `t2`, the DDL should not be ignored
case 2:
- rules = ['test.t1'], user create table `test`.`t2` in upstream, no table is created in downstream
- rename `t2` to `t1`, DDL executed with error but ignored, the following DMLs will meet error.

Reply:
for case 1:
we just skip the rename DDL

for case 2:
TiCDC can't handle this scenario, manual intervention is required